### PR TITLE
Update DLM pack/unpack to version 10

### PR DESCRIPTION
**Pull request by numaru**
The DLM files are now in version 10. Also an update was mandatory.
I unpacked all the files without any exception.

### CODE REVIEW

File: pydofus/dlm.py [hunk 1]
```diff
@@ -86,18 +86,35 @@ def read(self):
         self._obj["bottomNeighbourId"] = self.raw().read_int32()
         self._obj["leftNeighbourId"] = self.raw().read_int32()
         self._obj["rightNeighbourId"] = self.raw().read_int32()
-        self._obj["shadowBonusOnEntities"]  = self.raw().read_int32()
-
-        if self._obj["mapVersion"] >= 3:
+        self._obj["shadowBonusOnEntities"] = self.raw().read_uint32()
```
Comment (numaru) [inline pydofus/dlm.py:89]: Read as unsigned in the sources.

File: pydofus/dlm.py [hunk 2]
```diff
@@ -86,18 +86,35 @@ def read(self):
...
+
+        if self._obj["mapVersion"] >= 9:
+            read_color = self.raw().read_int32()
+            self._obj["backgroundAlpha"] = (read_color & 4278190080) >> 32
```
Comment (numaru) [inline pydofus/dlm.py:93]: I don't know why this one is shifted by 32 and not 24. It's done like that in the sources.

File: pydofus/dlm.py [hunk 3]
```diff
@@ -121,7 +138,7 @@ def read(self):
             fg.read()
             self._obj["foregroundsFixtures"].append(fg.getObj())
 
-        self._obj["unknown_1"] = self.raw().read_int32()
```
Comment (numaru) [inline pydofus/dlm.py:124]: This value is never used.

Comment (balciseri) [reply pydofus/dlm.py:124]: I believe you can't remove the line without letting the stream move forward.
Edit: I coudn't see it from this window, but you already took care about that in the source. Good job, nvm then.

File: pydofus/dlm.py [hunk 4]
```diff
@@ -364,14 +395,54 @@ def raw(self):
...
+            self._obj["nonWalkableDuringRP"] = (self._obj["losmov"] & 128) >> 7 == 1
+            self._obj["nonWalkableDuringFight"] = (self._obj["losmov"] & 4)
         self._obj["speed"] = self.raw().read_char()
-        self._obj["mapChangeData"] = self.raw().read_uchar()
+        self._obj["mapChangeData"] = self.raw().read_char()
 
         if self.mapVersion > 5:
             self._obj["moveZone"] = self.raw().read_uchar()
-        if self.mapVersion > 7:
+        if self.mapVersion > 7 and self.mapVersion < 9:
```
Comment (balciseri) [inline pydofus/dlm.py:445]: It's correct ( it matches ankama's source code) but it's funny :D

**Pull request merged**